arm-network/2016-09-01/swagger/virtualNetworkGateway.json

```diff
@@ -1132,7 +1374,39 @@
       },
       "description": "Response for the ListVirtualNetworkGateways API service call."
     },
-	"TunnelConnectionHealth": {
+    "BgpPeerStatusListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BgpPeerStatus"
+          },
+          "description": "List of BGP peers"
+        },
+        "nextLink": {
```

Comment:
Wow this is interesting! Since the model has nextLink property and a value property of type array, it looks that the operation that returns this model as the response is a pageable operations. However, this extension is not applied on the operation. At the same time this operation is marked as long running? Have never seen an operation that can be pageable and long running?

Couple of questions:
1. Is the operation to get the next page (GET on the link provided in the nextLink), also a long running operation?
2. If the user makes the initial request next time in a short span (say 5 minutes), will this operation still be long running, (i.e. will it still send a 202 as the initial response)? (Reason for asking: services usually cache the data for some time, if the same request is made in that window, then how is it long running?)